db/migrate/20170609100310_migrate_public_body_data.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+class MigratePublicBodyData < ActiveRecord::Migration
+  def up
+    PublicBody.find_each do |record|
```

Comment:
This isn't really needed.

We do need to figure out how to handle the internal admin authority, but it would be better to have a migration that just affects that, rather than every authority.